### ISSUE TITLE
Update to support Giants Foundry

### ISSUE THREAD

**Issue by aes5521** [TheStonedTurtle/banked-experience#84]
Unsure if this is still actively updated (extremely useful plugin!), but it would be great if we had the ability to calculate using Giants Foundry similar to here:
https://osrsportal.com/giants-foundry

**Comment by TheStonedTurtle** [comment]
Since you can use any combination of bars and any combination of the two to get 28 total bars the UI would be vastly different and doesn't really fit with the current plugins functionality. This plugin determines the xp based off a single banked item and not combinations of items. Also the exp from giants foundry can fluctuate due to how efficient you are at the minigame which is not the case for any other activity in the plugin.

TL;DR The UI requirements for this are not something I am willing to add to this plugin and calculating the xp would be quite difficult/inaccurate.

**Comment by christianlegge** [comment]
@TheStonedTurtle Would you be open to merging a PR if someone did figure out a nice way to add GF support and coded it? I understand not wanting to do it since it would be pretty different compared to how the plugin works for everything else but I think it would be really useful for a lot of people. (I can also see the argument that it should be its own plugin, but with so many people already using this one, I think it would be more helpful if it were incorporated here)

**Comment by TheStonedTurtle** [comment]
@christianlegge Sure, it's just gotta be a nice/clean implementation. The main issue with GF is that it you can do any combination of any bars, with any mould, and how well you make the sword matters. If you just want to add the "optimal" stuff then that's fine
